app/__init__.py

```diff
@@ -47,4 +47,19 @@ def remove_trailing_slash():
         if request.path != '/' and request.path.endswith('/'):
             return redirect(request.path[:-1], code=301)
 
+    @application.after_request
```

Comment:
dmutils should already be doing this for all apps: https://github.com/alphagov/digitalmarketplace-utils/blob/master/dmutils/__init__.py#L45